simpleLooping/binaryListToString.py

```diff
@@ -15,3 +15,16 @@
 # Ex:
 #   binaryListToString([1, 0, 1, 1])       # return "1101"
 #   binaryListToString([0, 1, 1, 0, 0, 1]) # return "100110"
+from typing import List
+
+def binaryListToString(lst: List[int]) -> str:
+    for num in lst:
```

Comment:
“num” is okay, but “digit” will be better because we check for each digit of a binary string.